src/plugins/event_annotation/public/event_annotation_service/service.tsx

```diff
@@ -48,6 +49,42 @@ export function getEventAnnotationService(): EventAnnotationServiceType {
             },
           ],
         };
+      } else if ('filter' in annotation) {
```

Comment:
Does it make sense to add an `isWhatever` utility here?

Reply:
This is the code that is never executed at this point (we still don't use `fetch_event_annotations` or `query_point_event_annotations` so I just wrote it for the types not to complain and it will have to be removed in the next PR so I didn't pay much attention to it. Added utility :)